ReactiveCocoa.podspec

```diff
@@ -24,7 +24,7 @@ Pod::Spec.new do |s|
   s.watchos.source_files = "ReactiveCocoa/WatchKit/*.{swift}"
   s.module_name = 'ReactiveCocoa'
 
-  s.dependency 'ReactiveSwift', '~> 4.0'
+  s.dependency 'ReactiveSwift', '~> 5.0'
```

Comment:
FYI ReactiveSwift 5.0 is not published to CocoaPods yet. Please wait for a while.

Reply:
That has been published.